### PR TITLE
remove populate_icon_from_ohloh

### PR DESCRIPTION
**Pull request by arindampradhan**
fix for #1524 
Do recommend changes if required.

### CODE REVIEW

File: mysite/search/migrations/0014_populate_icons_for_existing_projects_LATER.py [hunk 1]
```diff
@@ -29,11 +29,6 @@ def forwards(self, orm):
         # Populate icons for all existing projects.
         for orm_project in orm['search.project'].objects.all():
             model_project = Project.objects.get(pk=orm_project.pk)
-            print "Populating icon for", model_project
-            model_project.populate_icon_from_ohloh()
-            # We know this is unsafe.
-            # If this fails, it might be because
-            # populate_icon_from_ohloh doesn't exist anymore.
```
Comment (onceuponatimeforever) [inline mysite/search/migrations/0014_populate_icons_for_existing_projects_LATER.py]: Generally, you shouldn't modify anything in a `migrations` directory, as all the contents there are auto-generated by South for Django apps that are older than Django 1.7. Instead, use South to do a migration on the command line, which will add a migration file (to be git committed): http://south.readthedocs.org/en/latest/tutorial/part1.html

Comment (arindampradhan) [inline mysite/search/migrations/0014_populate_icons_for_existing_projects_LATER.py]: @onceuponatimeforever , on #1524, you can see why I changed the migrations in django, those are custom migrations and needed to be changed after I had removed ``populate_icon_from_ohloh()``  else would have caused  ``AttributeError`` .

Comment (onceuponatimeforever) [inline mysite/search/migrations/0014_populate_icons_for_existing_projects_LATER.py]: Can you run a migration to see if any new migration files get generated?

**Comment by onceuponatimeforever** [comment]
LGTM :apple: Looks like there are no additional migrations needed.

**Comment by arindampradhan** [comment]
Thanks :D  @onceuponatimeforever

**Pull request merged**